__tests__/app.test.js

```diff
@@ -111,6 +114,42 @@ describe('/api/articles', () => {
 
             })
     });
+});
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET 200 /api/articles/1/comments', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({ body: { comments } }) => {
+                expect(comments.length).toBe(11)
+                comments.forEach((comment) => {
+                    expect(typeof comment.comment_id).toBe('number')
+                    expect(typeof comment.votes).toBe('number')
+                    expect(typeof comment.created_at).toBe('string')
+                    expect(typeof comment.author).toBe('string')
+                    expect(typeof comment.body).toBe('string')
+                    expect(typeof comment.comment_id).toBe('number')
+                })
```

Comment:
Lovely check! For this one, all of these should have an article_id of 1, as that's what you've searched for. I think refelcting that in the test is a definite :)